recipes-debian/iperf2/iperf2/0003-fix_build_without_ipv6.patch

```diff
@@ -0,0 +1,21 @@
+--- a/src/Listener.cpp
```

Comment:
Please use the patch which was merged to the iperf2 upstream. You can use git-format-patch.

Reply:
It looks this patch back ported from "[fix compile when --disable-ipv6 is configured. Thanks to Gaetano Catalli](https://sourceforge.net/p/iperf2/code/ci/e7dc021b7f791256ef280bb25a398c03e4433c2a/)" patch. If so, please describe where patch come from.